setup.py

```diff
@@ -54,7 +54,7 @@ def enable_log_to_stdout(logname):
 
 setup(
     name=dist_name,
-    version="0.2",
+    version="0.3",
```

Comment:
New feature! Wooo!

Reply:
we may have forgotten to increment this previously, but the previous stable version was 0.3 and this new one will be 0.4

see https://github.com/fle-internal/kolibri-instant-schools-plugin/releases